packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -91,17 +107,21 @@ class MediaQueryData {
   ///    textScaleFactor defined for a [BuildContext].
   final double textScaleFactor;
 
-  /// The number of physical pixels on each side of the display rectangle into
-  /// which the application can render, but over which the operating system
-  /// will likely place system UI, such as the keyboard, that fully obscures
-  /// any content.
+  /// The parts of the display that are completely obscured by system UI,
+  /// typically by the device's keyboard.
+  ///
+  /// When a mobile device's keyboard is visible `viewInsets.bottom`
+  /// corresponds to the top of the keyboard.
+  ///
+  /// This value is independent of the [padding]: both values are
+  /// measured from the edges of the [MediaQuery] widget's bounds. The
+  /// bounds of the top level MediaQuery created by [WidgetsApp] are the
+  /// same as the window that contains the app.
```

Comment:
It might not be clear what 'window' is technically. If you bring it up, it might be worth qualifying specifically.

Reply:
OK: pointed out parenthetically that it's typically the screen.

Reply:
Bit of a nit but might not be true for add to app.